### DISCUSSION TITLE
Issues downloading a NuGet package version from Github Packages since 2022.11.30

### DISCUSSION THREAD

**Discussion by dfar-io** [community/community#40574] [Packages]
### Select Topic Area

Product Feedback

### Body

We're experiencing issues with running `dotnet restore` when trying to download versions for a package since 2022.11.30, so any packages pushed to Github Packages NuGet since then display in the package listing but can't be restored from a .NET project. When trying to restore, I'm provided the following when running `dotnet restore -v diag`:

```
GET https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json (TaskId:40)
                       GET https://nuget.pkg.github.com/xup-payments/download/xup.shared.library/index.json (TaskId:40)
                       NotFound https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json 30ms (TaskId:40)
                       GET https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json (TaskId:40)
                       NotFound https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json 27ms (TaskId:40)
                       OK https://nuget.pkg.github.com/xup-payments/download/xup.shared.library/index.json 520ms (TaskId:40)

Checking compatibility of packages on net6.0. (TaskId:40)
                     All packages and projects are compatible with net6.0. (TaskId:40)
                       NotFound https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json 27ms (TaskId:40)
                       GET https://api.nuget.org/v3-flatcontainer/xup.shared.library/index.json (TaskId:40)
                     Committing restore... (TaskId:40)
                     Generating MSBuild file /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/obj/Xup.Shared.Database.EF.csproj.nuget.g.props. (TaskId:40)
                     Generating MSBuild file /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/obj/Xup.Shared.Database.EF.csproj.nuget.g.targets. (TaskId:40)
                     Writing assets file to disk. Path: /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/obj/project.assets.json (TaskId:40)
                     Writing cache file to disk. Path: /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/obj/project.nuget.cache (TaskId:40)
                     Persisting dg to /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/obj/Xup.Shared.Database.EF.csproj.nuget.dgspec.json (TaskId:40)
                     Failed to restore /home/runner/work/Xup.Shared.Database/Xup.Shared.Database/Xup.Shared.Database.EF/Xup.Shared.Database.EF.csproj (in 2 sec). (TaskId:40)
```

Is there recently changed configuration I need to make to allow for restoring correctly?

Some information on observations:

* I am able to see the deployed versions in package listing via https://github.com/xup-payments/Xup.Shared/pkgs/nuget/XUP.Shared.Library and https://nuget.pkg.github.com/xup-payments/download/xup.shared.library/index.json
* As these deployments are driven by Github Release creation, the releases no longer include the `.nupkg` files.
* When making an attempt to restore, the download count in the package listing increases like so:

![image](https://user-images.githubusercontent.com/564106/205147243-50c804a6-850a-45fe-8153-8c2e0b88a899.png)

**Comment by ossanna16** [comment]
Hi there @dfar-io and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by dfar-io** [reply]
Ended up solving this by having a mismatching package ID inside `.csproj`
